### ISSUE TITLE
Ctrl+/ does not comment anymore

### ISSUE THREAD

**Issue by AbdealiJK** [jupyterlab/jupyterlab#1901]
I remember in some older version of JupyterLab doing a `Command+#` used to comment a line and uncomment it if it was already commented.
Jupyter Notebook does the same.

When using master, I notice that this feature does not work anymore and does nothing.

Not sure if it was removed for a reason, but was not able to find any issue or commit that explicitly removes this feature.

**Comment by AbdealiJK** [comment]
This is happening because of the commit https://github.com/jupyterlab/jupyterlab/commit/b18ca0b11f8f4b48bce3aefdf578f4584bd21bcb

Where the default options for the factory is being handled in the constructor, but the function `newInlineEditor` is being passed around:
```python
  let editorFactory = editorServices.factoryService.newInlineEditor;
  let contentFactory = new NotebookPanel.ContentFactory({ editorFactory });
```
When it's passed around as a function, the `this` keyword becomes an empty object and so none of the default options are being used.

In a nutshell: The function was being used as a static function, but now it has state so it's buggy.

**Comment by erichiller** [comment]
I'm encountered this in the 0.27 pypi build. Ctrl + / does not work for commenting lines. Should it?

**Comment by blink1073** [comment]
Hi @erichiller, it should work if select `Editor -> Key Map -> Sublime Text`.

**Comment by erichiller** [comment]
@blink1073  thanks much, I did get commenting working, but now shortcuts such as ctrl-d for line deletion are removed/changed. Is there any way to, or plans to add commenting hot keys or otherwise modify the default keymap?

**Comment by blink1073** [comment]
@erichiller, there are no concrete plans on how to handle modifying keyboard shortcuts in the editor, please feel free to open an issue.

**Comment by adrienrenaud** [comment]
I can't find any shortcut to toggle comments in a python notebook cell in JupyterLab 0.34.9. This is probably one of the most useful features in a text editor.
Could the Jupyter Team give a clear solution to have Ctrl+/ toggling comments?

**Comment by blink1073** [comment]
Hi @erichiller, is your keyboard using a layout other than US English?  If so, it is a known problem, see https://github.com/jupyterlab/jupyterlab/issues/5164.

**Comment by adrienrenaud** [comment]
Thanks @blink1073 for your reply. I'm indeed using a french keyboard. But the Ctrl+/ is working on another laptop with french keyboard. Any workaround (including using another shortcut than Ctrl+/)?

**Comment by blink1073** [comment]
Not that I am aware of, currently.
